src/client_backend/openai/http_client.h

```diff
@@ -142,7 +147,6 @@ class HttpClient {
   // Note that this function does not block
   void Send(CURL* handle, std::unique_ptr<HttpRequest>&& request);
 
- protected:
```

Comment:
Is there a reason we no longer want this information to be protected?

Reply:
I'm guessing it's to access `total_input_byte_size_` from `ChatCompletionClient::PrintCurlCommand`. It'd probably be better to add a public accessor for it instead.

Reply:
Aren't they in protected ? We have defined at line 140 and all the others that follow are protected right? 
I removed thinking it's a redundant declaration.

Reply:
Oh, derp, totally missed the fact this `protected:` here is redundant. Proceed ^_^

Reply:
Is it why I removed the line 145? I think it's redundant and that's why I removed. we have one protected in line 140 and everything that follows till line 171 is protected?

Reply:
Yep, our bad, we just haven't been paying attention.